recipes/nlohmann_fifo_map/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "nlohmann_fifo_map" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "bb2df8633d6d1a31528191a930d1ee2469ed31fb682fa53a1044962c8f7a1a2e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/nlohmann/fifo_map/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: mkdir "%LIBRARY_INC%\\nlohmann" && move "%SRC_DIR%\\src\\*.hpp" "%LIBRARY_INC%\\nlohmann"  # [win]
+  script: mkdir -p ${PREFIX}/include/nlohmann && mv $SRC_DIR/src/*.hpp --target-dir=${PREFIX}/include/nlohmann  # [unix]
```

Comment:
I believe we can change this recipe to be ``noarch: generic`` as it is header-only and you are not dependant on platform this can be ``noarch: generic``. What do you think?
So we will just need the second line, for ``noarch`` we just kick the linux CI to generate the package

Reply:
I just remember that on windows we use ``Library`` before the include, so we cannot transform that as noarch even though it is independent of platform